systems/plants/@RigidBodyManipulator/manipulatorDynamics.m

```diff
@@ -31,6 +25,11 @@
   [f_ext, B] = computeExternalForcesAndInputMatrix(obj, q, v);
 end
 
+options.use_mex = use_mex;
+options.compute_gradients = compute_gradients;
+options.compute_JdotV = true;
```

Comment:
still looks strange to me that you'd have to compute JdotV in order to get H or C.